### ISSUE TITLE
Add DeepAlpha - AI trading with deep learning ensemble

### ISSUE THREAD

**Issue by stefanoviana** [aymericdamien/TopDeepLearning#36]
Hi! I would like to suggest adding [DeepAlpha](https://github.com/stefanoviana/deepalpha) to this list.

**What is DeepAlpha?**
AI crypto trading bot with 70.9% walk-forward validated accuracy. XGBoost + LightGBM ensemble with 72 features. MIT license.

**Suggested entry:**
```
- [DeepAlpha](https://github.com/stefanoviana/deepalpha) - AI crypto trading bot with 70.9% walk-forward validated accuracy. XGBoost + LightGBM ensemble with 72 features. MIT license.
```

Thank you!
